airflow/providers/google/cloud/example_dags/example_gcs_to_bigquery.py

```diff
@@ -29,7 +29,7 @@
 }
 
 dag = models.DAG(
-    dag_id='example_gcs_to_bq_operator', default_args=args,
+    dag_id='example_gcs_to_bigquery_operator', default_args=args,
     schedule_interval=None, tags=['example'])
 
 create_test_dataset = BashOperator(
```

Comment:
Unfortunately, this command will fail if you are not logged in to gcloud. I'm working to fix it.
https://github.com/apache/airflow/pull/8432
Can you run this command in setUp/tearDown or use dedicated operators?

Reply:
Does breeze log you in on google cloud CLI? All I did was provide the credentials. Using the operators seems like a better and cleaner solution indeed. Will rework it